proto/minder/v1/minder.proto

```diff
@@ -2259,12 +2277,30 @@ message Profile {
     //
     // DNS_STR = "[a-zA-Z0-9](?[-a-zA-Z0-9]{0,61}[a-zA-Z0-9])?"
     // ($DNS_STR:)?$DNS_STR
-    repeated string labels = 12;
+    repeated string labels = 12 [
+        (buf.validate.field) = {
+            repeated: {
+                items: {
+                    string: {
+                        pattern: "^([a-zA-Z0-9]([-a-zA-Z0-9]{0,61}[a-zA-Z0-9])?:)?[a-zA-Z0-9]([-a-zA-Z0-9]{0,61}[a-zA-Z0-9])?$"
+                    }
```

Comment:
Not sure if you want to keep or remove the preceeding comment.  Maybe keep until we can generate these in the OpenAPI doc?

Reply:
Let's leave it for now 👍